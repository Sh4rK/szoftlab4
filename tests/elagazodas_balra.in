loadMap elagazodas.map
loadMission one_enemy.mission
step 2
listEnemies
exit