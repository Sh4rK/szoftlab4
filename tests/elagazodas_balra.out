1 100 (0;0,33)
